.github/workflows/docker-connector-image-publishing.yml

```diff
@@ -45,6 +45,9 @@ jobs:
     name: Build and Publish ${{ github.event.inputs.connector-type }} Base Image
     runs-on: ubuntu-latest
     if: ${{ github.event.inputs.image-type == 'base' }}
+    environment:
+      name: ${{ github.event.inputs.repository-root == 'docker.io/airbyte' && 'hub.docker.com/r/airbyte' || github.event.inputs.repository-root }}
+      url: https://${{ github.event.inputs.repository-root == 'docker.io/airbyte' && 'hub.docker.com/r/airbyte' || github.event.inputs.repository-root }}/${{ github.event.inputs.connector-type }}-connector-base
```

Comment:
This is a bit messy because we have to swap 'docker.io/airbyte' with 'hub.docker.com/r/airbyte' in the URL for dockerhub to make it work in https, whereas the 'ghcr.io/airbytehq' URL works fine as is.